cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CompileTask.java

```diff
@@ -239,6 +269,13 @@ public void execute(Project project) {
         }
     }
 
+    private String getDocumentPath(ModuleName moduleName, String documentName) {
+        if (moduleName.isDefaultModuleName()) {
+            return documentName;
+        }
+        return "modules" + "/" + moduleName.moduleNamePart() + "/" + documentName;
```

Comment:
The directory separator is depend on the OS. We cannot directly use `\`. wdyt?

Reply:
Agreed, will test out how paths are returned from the diagnosticAPI on windows.

Reply:
You can use the `Paths.get` to join paths.

Reply:
Thanks!